.NET/Microsoft.Recognizers.Definitions.Common/Dutch/DateTimeDefinitions.cs

```diff
@@ -128,6 +128,7 @@ public static class DateTimeDefinitions
       public const string DatePreposition = @"\b(op(\s+de)?)";
       public static readonly string DateExtractorYearTermRegex = $@"(\s+|\s*,\s*){DateYearRegex}";
       public static readonly string DateExtractor1 = $@"\b({WeekDayRegex}\s*[,-]?\s*)?(({MonthRegex}(\.)?\s*[/\\.,-]?\s*{DayRegex})|(\({MonthRegex}\s*[-.]\s*{DayRegex}\)))(\s*\(\s*{WeekDayRegex}\s*\))?({DateExtractorYearTermRegex}\b)?";
+      public static readonly string DateExtractor2 = $@"\b({WeekDayRegex}\s*[,-]?\s*)?(?<!\d[.,])(({MonthRegex}[\.]?\s*[/\\.,-]?)\s*({DayRegex}\s*[/\\.,-]?\s*{BaseDateTime.FourDigitYearRegex}|{BaseDateTime.FourDigitYearRegex}\s*[/\\.,-]?\s*{DayRegex})|({DayRegex}\s*[/\\.,-]?)\s*({MonthRegex}[\.]?\s*[/\\.,-]?\s*{BaseDateTime.FourDigitYearRegex}|{BaseDateTime.FourDigitYearRegex}\s*[/\\.,-]?\s*{MonthRegex}[\.]?)|({BaseDateTime.FourDigitYearRegex}\s*[/\\.,-]?)\s*({DayRegex}\s*[/\\.,-]?\s*{MonthRegex}[\.]?|{MonthRegex}[\.]?\s*[/\\.,-]?\s*{DayRegex}))\b";

```

Comment:
Do we really need to add a new regex for all extra cases? I'm a bit worried about performance impact as it's one more regex to execute and it looks long and complex.
Could you run some experiments with different input (repeated dates will hit the cache) and try to measure the impact? Running English only is ok.

Reply:
For ease of reading and  maintenance, I'd also prefer if this new regex has a consistent name across cultures and if it's name is more descriptive.

Reply:
> Do we really need to add a new regex for all extra cases?

Not necessarely, I followed this approach because it is more readable and easier to port to other cultures. 
I will run the experiments and in case try to merge the unsupported patterns with the existing regexes.

Reply:
The impact of the new regex is noticeable only when considering long input strings and in this case it appears to be more performant to merge the new patterns in the existing regexes. 
So, I removed the new regex and modified the existing regexes (only in English and Spanish for now).